ElectronClient/gui/NoteContentPropertiesDialog.tsx

```diff
@@ -54,28 +93,69 @@ export default function NoteContentPropertiesDialog(props:NoteContentPropertiesD
 		props.onClose();
 	};
 
-	const createItemField = (key: string, value: number) => {
-		const labelComp = <label style={Object.assign({}, theme.textStyle, theme.controlBoxLabel)}>{keyToLabel[key]}</label>;
-		const controlComp = <div style={Object.assign({}, theme.textStyle, theme.controlBoxValue)}>{value}</div>;
+	const labelCompStyle = {
+		...theme.textStyle,
+		fontWeight: 'bold',
+		width: '10em',
+	};
+
+	const controlCompColumnWidth = '7em';
+
+	const controlCompStyle = {
+		...theme.textStyle,
+		width: controlCompColumnWidth,
+		textAlign: 'center',
+	};
+
+	const createTableBodyRow = (key: string, value: number, strippedValue: number) => {
+		const labelComp = <td style={labelCompStyle}>{keyToLabel[key]}</td>;
+		const controlComp = <td style={controlCompStyle}>{value}</td>;
+		const strippedControlComp = <td style={controlCompStyle}>{strippedValue}</td>;
 
 		return (
-			<div key={key} style={theme.controlBox} className="note-text-property-box">{labelComp}{controlComp}</div>
+			<tr key={key}>{labelComp}{controlComp}{strippedControlComp}</tr>
 		);
 	};
 
+	const tableHeaderStyle = {
+		...theme.textStyle,
+		width: controlCompColumnWidth,
+		textAlign: 'center',
+	};
+
+	const tableHeader = (
+		<tr>
+			<th style={tableHeaderStyle}></th>
+			<th style={tableHeaderStyle}>{_('Raw Source')}</th>
+			<th style={tableHeaderStyle}>{_('Markdown-stripped source')}</th>
+		</tr>
+	);
+
 	if (textProperties) {
```

Comment:
This should always be set, so please remove the conditional.

Reply:
Removed as you instructed